src/openms/source/CHEMISTRY/EnzymaticDigestionLogModel.cpp

```diff
@@ -161,7 +161,7 @@ namespace OpenMS
   {
     SignedSize count = 1;
     AASequence::ConstIterator iterator = protein.begin();
-    while (nextCleavageSite_(protein, iterator), iterator != protein.end())
+    while (static_cast<void>(nextCleavageSite_(protein, iterator)), iterator != protein.end())
```

Comment:
hmm what did he complain about here?

Reply:
possible misuse of comma operator here [-Wcomma]

Reply:
ok what you could also do to make this more readable is to write:
```
nextCleavageSite_(protein, iterator);
while (iterator != protein.end())
{
// as before...
nextCleavageSite_(protein, iterator);
}
```
that is the doing the same thing and don't uses the comma operator in while loops

Reply:
ok, thanks!

Reply:
How would this function look then? 
```
Size EnzymaticDigestionLogModel::peptideCount(const AASequence& protein)
  {
    SignedSize count = 1;
    AASequence::ConstIterator iterator = protein.begin();
    while (static_cast<void>(nextCleavageSite_(protein, iterator)), iterator != protein.end())
    {
      ++count;
    }
    Size sum = count;
    return sum;
  }
```
Like this?
```
Size EnzymaticDigestionLogModel::peptideCount(const AASequence& protein)
  {
    SignedSize count = 1;
    AASequence::ConstIterator iterator = protein.begin();
    while (iterator != protein.end())
    {
      nextCleavageSite_(protein, iterator)
      ++count;
    }
    Size sum = count;
    return sum;
  }
```

Reply:
nextCleavageSite_(protein, iterator);
while (iterator != protein.end())
{
++count;
nextCleavageSite_(protein, iterator);
}